.gitignore

```diff
@@ -17,3 +17,4 @@ build.py
 .noseids
 *.db
 static
+celerybeat-schedule
```

Comment:
I'd add this one to `.dockerignore` as well to avoid including the celery db into docker images.